tests/integration/conftest.py

```diff
@@ -183,3 +184,17 @@ def _track_notification(*args, **kwargs):  # type: ignore
     )
 
     yield calls
+
+
+@pytest.fixture()
+def authenticated_client(
+    anonymous_client: FlaskClient, factories: _Factories, request: FixtureRequest
+) -> Generator[FlaskClient, None, None]:
+    email_mark = request.node.get_closest_marker("authenticate_as")
```

Comment:
Splitting out the user who will be authenticated as a marker feels nice as permissions and options grow!